dashboard/app/controllers/user_school_infos_controller.rb

```diff
@@ -0,0 +1,13 @@
+class UserSchoolInfosController < ApplicationController
+  def update
+    user_school_info = UserSchoolInfo.find([:id])
```

Comment:
i don't think we can access id like it is here -- i think you meant `params[:id]`